src/scripts/admin/blocks/collapse/container/index.js

```diff
@@ -0,0 +1,62 @@
+import { __ } from '@wordpress/i18n';
+import { registerBlockType } from '@wordpress/blocks';
+import { InnerBlocks } from '@wordpress/block-editor';
+import { deprecated } from './deprecated';
+import { attributes } from './attributes';
+
+registerBlockType('bcgov/collapse', {

```

Comment:
might want to change the namespace, i'm guessing is the original one..  And keep consistent for all blocks you might create.